website/frontend/templates/docs/build-windows.html

```diff
@@ -6,113 +6,192 @@
         </li>
       </ul>
 
-      <h3 id="visual-cpp">Visual C++ 2008 Express</h3>
+      <h1 id="running_picard_from_source">Running Picard From Source</h1>
+
+      <h3 id="chromaprint">Chromaprint</h3>
+      
+      <p>
+        If you want to run the Acoustid Scan function when running Picard from source, then you need to install
+        the Chromaprint fpcalc program into your Picard source directory.
+      </p>
 
       <ul>
-        <li> <a rel="nofollow" href="http://go.microsoft.com/?linkid=7729279">http://go.microsoft.com/?linkid=7729279</a>
+        <li>
+          Download <a rel="nofollow" href="https://github.com/acoustid/chromaprint/releases/download/v1.4.2/chromaprint-fpcalc-1.4.2-windows-x86_64.zip">the windows zip file</a> 
+          from <a rel="nofollow" href="http://acoustid.org/chromaprint">http://acoustid.org/chromaprint</a>
         </li>
-        <li>Download and install "vcsetup.exe"
+        <li>
+          Unzip "chromaprint-fpcalc-1.1-win-i86.zip" and put "fpcalc.exe" to the picard subdirectory 
+          of your Picard source code directory
         </li>
       </ul>
 
-      <h3 id="chromaprint">Chromaprint</h3>
+      <h3 id="gettext">gettext</h3>
+      
+      <p> 
+        Picard uses gettext for internationalisation (i.e. for Picard to appear in your local language). 
+        To build the language mappings you need to install gettext.
+      </p> 
 
       <ul>
-        <li> <a rel="nofollow" href="http://acoustid.org/chromaprint">http://acoustid.org/chromaprint</a>
+        <li>
+          Download and install <a rel="nofollow" href="https://downloads.sourceforge.net/project/gnuwin32/gettext/0.14.4/gettext-0.14.4.exe">gettext</a> 
+          from <a rel="nofollow" target="_blank" href="http://gnuwin32.sourceforge.net/packages/gettext.htm">http://gnuwin32.sourceforge.net/packages/gettext.htm</a>.
+        </li>
+        <li>
+          Add the gettext folder to&nbsp;%PATH% (default <i>C:\Program&nbsp;Files\GnuWin32\bin</i> or <i>C:\Program&nbsp;Files&nbsp;(x86)\GnuWin32\bin</i>)
         </li>
-        <li>Download "chromaprint-fpcalc-1.1-win-i86.zip" and put "fpcalc.exe" to the Picard source code
-          directory
+        <li>
+          From the Picard source directory run
+          <pre>py setup.py build_locales -i</pre>
         </li>
       </ul>
 
-      <h1 id="running_picard_from_sources">Running Picard From Sources</h1>
-
-      <p>
-        Before you can run Picard from sources, you need to build the C extension. Start the VS console using
-        "Visual Studio 2008 Command Prompt", go to the source directory and run these commands:
-      </p>
-
-      <pre>
-        python setup.py build_ext -i
-        python setup.py build_locales -i</pre>
-
       <p>
         After you have done this, you can run Picard directly from the sources using:
       </p>
 
-      <pre>
-        python tagger.py</pre>
+      <pre>py tagger.py</pre>
```

Comment:
Yes it does!! You can either run "py -2" or "py -3" or let "py" look at the first line in the .py file to see what version it should run.